salt/modules/aix_group.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+'''
+Manage groups on Solaris
+
+.. important::
+    If you feel that Salt should be using this module to manage groups on a
+    minion, and it is using a different module (or gives an error similar to
+    *'group.info' is not available*), see :ref:`here
+    <module-provider-override>`.
+'''
+from __future__ import absolute_import
+
+# Import python libs
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+try:
+    import grp
+except ImportError:
```

Comment:
I don't understand why this would be necessary. Isn't `grp` a part of the stdlib? If for some crazy reason it isn't there, functions in this module will be broken anyway.